src/components/forms/SmallGrantsForm.tsx

```diff
@@ -0,0 +1,1190 @@
+                  _placeholder={{ fontSize: 'input' }}
+                  color='brand.paragraph'
+                  fontSize='input'
+                  h='150px'
+                  mt={3}
+                />
+              </FormControl>
+
+              <FormControl id='project-additional-info-control' mb={8}>
+                <FormLabel htmlFor='projectAdditionalInfo' mb={1}>
+                  <PageText fontSize='input'>Anything else you&apos;d like to share?</PageText>
+                </FormLabel>
+
+                <PageText as='small' fontSize='helpText' color='brand.helpText'>
+                  Is there anything we should know about that hasn&apos;t been covered by the
+                  questions above? You also have the option to link any supporting documents or
+                  relevant sites here.
+                </PageText>
+
+                <Textarea
+                  id='project-additional-info'
+                  // TODO: change this when input validation is added
+                  // value={''}
+                  // onChange={() => {}}
+                  bg='white'
+                  borderRadius={0}
+                  borderColor='brand.border'
+                  _placeholder={{ fontSize: 'input' }}
+                  color='brand.paragraph'
+                  fontSize='input'
+                  h='150px'
+                  mt={3}
+                />
+              </FormControl>
+            </>
+          )}
+
+        {(projectCategory as ProjectCategory).value === COMMUNITY_EVENT && (
+          <>
+            <FormControl id='event-name-control' isRequired mb={8}>
+              <FormLabel htmlFor='eventName' mb={1}>
+                <PageText display='inline' fontSize='input'>
+                  Event name
+                </PageText>
+              </FormLabel>
+
+              <PageText as='small' fontSize='helpText' color='brand.helpText'>
+                What&apos;s the official title of your event?
+              </PageText>
+
+              <Input
+                id='event-name'
+                type='text'
+                bg='white'
+                borderRadius={0}
+                borderColor='brand.border'
+                h='56px'
+                _placeholder={{ fontSize: 'input' }}
+                color='brand.paragraph'
+                fontSize='input'
+                mt={3}
+              />
+            </FormControl>
+
+            <FormControl id='event-date-control' isRequired mb={8}>
+              <FormLabel htmlFor='eventDate' mb={1}>
+                <PageText display='inline' fontSize='input'>
+                  Event date
+                </PageText>
+              </FormLabel>
+
+              <PageText as='small' fontSize='helpText' color='brand.helpText'>
+                Please enter the first date of your event (MM/DD/YYYY)
+              </PageText>
+
+              {/* TODO: convert date to 'MM/DD/YYYY' format before submitting */}
+              <Input
```

Comment:
I guess we will need to require that the date is not old as well

Reply:
Sure, I'll add that validation later, maybe we'd need to switch to `react-datepicker` instead of the native date input to ease that validations.

Reply:
Cool